### PR TITLE
DEVOPS-987-10: aha-circle-docker container

### PR DESCRIPTION
**Pull request by slamotte**
Update base image. Replace deprecated key service.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -46,7 +46,7 @@ RUN cat /tmp/chrome.pub | apt-key add -
 RUN echo "deb https://dl.google.com/linux/chrome/deb/ stable main" > /etc/apt/sources.list.d/google.list
 
 # Update keys
-RUN apt-key adv --refresh-keys --keyserver ha.pool.sks-keyservers.net
+RUN apt-key adv --refresh-keys --keyserver keys.openpgp.org
```
Comment (slamotte) [inline Dockerfile:49]: `pool.sks-keyservers.net` has been out of service since April 2021.

Comment (slamotte) [reply Dockerfile:49]: @joseph-aha Any concerns with this change?

Comment (shanet) [reply Dockerfile:49]: Why are keyservers always such a pain? A reliable "Keyservers-as-a-service" sounds like a great startup idea.

**Review by shanet — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -46,7 +46,7 @@ RUN cat /tmp/chrome.pub | apt-key add -
 RUN echo "deb https://dl.google.com/linux/chrome/deb/ stable main" > /etc/apt/sources.list.d/google.list
 
 # Update keys
-RUN apt-key adv --refresh-keys --keyserver ha.pool.sks-keyservers.net
+RUN apt-key adv --refresh-keys
```
Comment (slamotte) [inline Dockerfile:49]: Specifying the keyserver is actually not necessary and will default to `keys.openpgp.org`

**Pull request merged**